spec/cypress/integration/StartPage.feature

```diff
@@ -7,3 +7,9 @@ Feature: Start Page
     And the page should be accessible
     And percy should be sent snapshot
 
+  Scenario: Should have cookies link
+    When I am on "start" page
+    Then "footer" should contain "Cookies"
+    And the page should be accessible
+    And percy should be sent snapshot
```

Comment:
I think we can add the footer check to the previous test. Which already checks whether the page is accessible and sends a snapshot to percy :)

Reply:
It feels kind of unrelated to that (and anything in this file) - maybe somewhere in Cookies.feature instead?

Reply:
Yeah, happy to move it there. I thought Cookies.feature was only for the cookie page but looks like other tests in there go from the start page.